### ISSUE TITLE
Kubernetes plugin question

### ISSUE THREAD

**Issue by aaronmell** [Netflix/lemur#1790]
I added a certificate and a Kubernetes endpoint. The certificate doesn't show up in Kubernetes and I can't seem to find any logs why.  Where can I find information about why it failed to place the certs in kubernetes? I'm running the app as docker containers and attempting to load the cert into EKS

**Comment by kevgliss** [comment]
Are you using the UI or API? Is there anything in `lemur.log`?

**Comment by kevgliss** [comment]
cc @mik373

**Comment by wfhartford** [comment]
I am having the same experience. Lemur is running in a Kubernetes cluster, trying to deploy to the same cluster. The SFTP destination is working, but I get no feedback at all from the Kuberentes destination. There are no messages about it in the pod logs or the lemur.log file in the container. I'm interacting with lemur through the UI.

**Comment by wfhartford** [comment]
I took a closer look at the Kubernetes plugin code and there are some pretty serious issues with it, mostly related to incompatible types. I'm not a python expert, but my guess is that the plugin worked in python 2 but fails in python 3. I've made a few changes that make the plugin work. They're pretty ugly at this stage, but I'll clean up my fixes and submit a PR.

There are a few functional changes I would like to make as well. Would a separate PR which makes the following changes be accepted?
* Allow configuration of Kubernetes certificate and token as file paths, better supporting lemur running inside Kubernetes.
* Default the Kubernetes namespace to the namespace Lemur is running in if it is running in Kubernetes.
* Configurable deployment of private keys so that the plugin only deploys certificates by default.
